tests/framework/Optimizers/SimulatedAnnealing/BoltzmanSA/test_simulatedAnnealing.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="debug">
+  <TestInfo>
+    <name>framework/Optimizers.SimulatedAnnealing</name>
```

Comment:
as above

Reply:
Done. But is the name used anywhere else? So If I changed the name will that affect anything else?
I changed it to: framework/Optimizers.SimulatedAnnealing.<cooling>.<EggHolder if exists>. is that OK?

Reply:
The path should be the full path of the test, right? In this case, it looks like `framework/Optimizers/SimulatedAnnealing/BoltmanSA/????` where `????` is whatever you named in in the `tests` file.

Reply:
@PaulTalbot-INL and @alfoa 
So if I called it BoltzmanSA in the tests then the name here should be:
`framework/Optimizers/SimulatedAnnealing/BoltzmanSA/BoltzmanSA` ?
I guess I am not sure of the difference between the `/` and the `.` in `framework/Optimizers.SimulatedAnnealing` when to use `/` and when to use `.`